### ISSUE TITLE
NVDA do not announce the state of check box selection

### ISSUE THREAD

**Issue by WiproCRMAccessibility** [nvaccess/nvda#9191]
Open attached HTML. Start NVDA and then follow these steps.
1) Navigate to item and click/press Space, The aria-selected flag is updating. But NVDA is not reading that as selected/non-selected.

[ariaSelected.zip](https://github.com/nvaccess/nvda/files/2786547/ariaSelected.zip)

**Comment by cabeln** [comment]
The observation was made on Edge browser.

**Comment by Adriani90** [comment]
This works as expected in Edge Cromium 81. Since this will be distributed broadly through the Windows Update 20 H1, I will close this issue as works for me.

**Comment by Adriani90** [comment]
Adding appropriate label since this is probably still reproducible in Edge Spartan and UIA related environments.
